src/open_inwoner/scss/components/Faq/_Faq.scss

```diff
@@ -7,6 +7,10 @@
     margin-bottom: var(--spacing-large);
   }
 
+  .h3 {
+    margin: 0;
+  }
```

Comment:
Is this necessary? I didn't check everywhere, but at least under `onderwerpen/producten/logeeropvang/` the `h3` element has no margin all-around.

Reply:
@pi-sigma you are inspecting the wrong H3 then... this is the new H3 we need inside the FAQ - so it needs to get rid of its default HTML margins.